internal/wazeroir/operations.go

```diff
@@ -819,27 +746,32 @@ func (l LabelID) FrameID() int {
 
 // ID returns the LabelID for this Label.
 func (l Label) ID() (id LabelID) {
-	id = LabelID(l.Kind) | LabelID(l.FrameID)<<32
+	id = NewLabelID(l.Kind, l.FrameID)
 	return
 }
 
+func NewLabelID(kind LabelKind, frameID uint32) LabelID {
+	return LabelID(kind) | LabelID(frameID)<<32
+}
+
 // String implements fmt.Stringer.
-func (l Label) String() (ret string) {
-	switch l.Kind {
+func (lid LabelID) String() (ret string) {
```

Comment:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/13513977/229645975-d7d54ff3-7003-4677-a78d-0cafd5f7ce37.png">

```suggestion
func (l LabelID) String() (ret string) {
```